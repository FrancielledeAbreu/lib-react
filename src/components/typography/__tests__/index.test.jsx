import React from 'react'
import Typography from '../index'
import renderer from 'react-test-renderer';

it("renders correctly", () => {
    const tree = renderer.create(<Typography value="" label/>).toJSON()

    expect(tree).toMatchSnapshot()
})