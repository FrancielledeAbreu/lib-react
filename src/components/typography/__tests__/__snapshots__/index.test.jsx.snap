// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot does not render higher than more six 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h6>
    Teste
  </h6>
</div>
`;

exports[`Snapshot renders correctly when the value is 1 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h1>
    Teste
  </h1>
</div>
`;

exports[`Snapshot renders correctly when the value is 2 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h2>
    Teste
  </h2>
</div>
`;

exports[`Snapshot renders correctly when the value is 3 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h3>
    Teste
  </h3>
</div>
`;

exports[`Snapshot renders correctly when the value is 4 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h4>
    Teste
  </h4>
</div>
`;

exports[`Snapshot renders correctly when the value is 5 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h5>
    Teste
  </h5>
</div>
`;

exports[`Snapshot renders correctly when the value is 6 1`] = `
<div
  className="sc-bdnylx jIjVkI"
>
  <h6>
    Teste
  </h6>
</div>
`;
